<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-SDJM71R" failures="1" tests="3" name="Test" time="14.132" errors="0" timestamp="2025-02-12T21:39:45 GMT+06:00">
  <testcase classname="day21.OrangeHRM" name="logoTest" time="3.063"/>
  <testcase classname="day21.OrangeHRM" name="appURL" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day21.OrangeHRM.appURL(OrangeHRM.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- appURL -->
  <testcase classname="day21.OrangeHRM" name="homePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- homePageTitle -->
</testsuite> <!-- Test -->
